src/home_robot/home_robot/agent/objectnav_agent/oracle_nav_agent.py

```diff
@@ -0,0 +1,101 @@
+import os
+from typing import Union
+
+import numpy as np
+from habitat.core.agent import Agent
+from habitat.sims.habitat_simulator.actions import HabitatSimActions
+from habitat.tasks.nav.shortest_path_follower import ShortestPathFollower
+
+from home_robot.core.interfaces import DiscreteNavigationAction
+
+# Quiet the Habitat simulator logging
+os.environ["MAGNUM_LOG"] = "quiet"
+os.environ["HABITAT_SIM_LOG"] = "quiet"
+
+
+def euclidean(v1, v2):
+    return sum((p - q) ** 2 for p, q in zip(v1, v2)) ** 0.5
+
+
+class ShortestPathFollowerAgent(Agent):
```

Comment:
@prasoonvarshney  can you move this into home_robot_sim?

Reply:
The whole class, i.e., everything related to shortest path?